libraries/lib-wave-track/WaveTrack.cpp

```diff
@@ -2596,10 +2663,57 @@ void WaveTrack::GetEnvelopeValues(
 // latter clip is returned.
 WaveClip* WaveTrack::GetClipAtTime(double time)
 {
+   return const_cast<WaveClip*>(
+      const_cast<const WaveTrack&>(*this).GetClipAtTime(time));
+}
+
+const WaveClip* WaveTrack::GetNeighbourClip(
```

Comment:
Generally we prefer American spellings, "neighbor"

Reply:
Me too, since Audacity is originally American. That one slipped. Thanks.

Reply:
... In fact I'm renaming this "GetNextClip". "Neighbor" could be confused with "adjacent".